include/MarlinTrk/IMarlinTrack.h

```diff
@@ -8,14 +8,25 @@
 #include "EVENT/TrackerHit.h"
 #include "IMPL/TrackStateImpl.h"
 
-#include "gearimpl/Vector3D.h"
-
+#include "DDSurfaces/Vector3D.h"
 
 #include <exception>
 #include <string>
 
+
+// for now provide wrapers using gear::Vector3D for backwards compatibility
+// if this is removed, we do not use Gear anywhere in MarlinTrk
+#define PROVIDE_BACKWARD_GEAR_WRAPPERS 1
```

Comment:
Better make this steerable via cmake so the it can be turned on and off also for MarlinTrkProcessors et al.